Backend/src/server.test.ts

```diff
@@ -0,0 +1,11 @@
+import supertest from 'supertest';
+import { server } from './server';
+
+const request = supertest(server);
+
+it('Gets / endpoint', async () => {
```

Comment:
Every test file should have a `describe('<Class to test>', () => {` block which contains the tests